src/lib/apis/fetch.ts

```diff
@@ -23,7 +24,18 @@ export default (url, options = {}) => {
     delete opts.userAgent
     opts.headers = assign({}, { "User-Agent": userAgent }, opts.headers)
 
-    request(url, opts, (err, response) => {
+    const { method } = opts
+    let cleanedUrl: string = url
+    if (method === "PUT" || method === "POST") {
+      // Move query params out of url and place into `body`.
+      const params = parse(cleanedUrl)
+      cleanedUrl = cleanedUrl.split("?")[0]
+
+      opts.body = params
+      opts.json = true
+    }
+
+    request(cleanedUrl, opts, (err, response) => {
```

Comment:
Is it possible to add a test for these changes?

Reply:
Of course! Will add.